lib/grpcApi/BlockExecutionState.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Store contracts during block execution process
+ */
+class BlockExecutionState {
```

Comment:
From Drive perspective, it's transaction execution due to `startTransaction` and `commitTransaction` endpoint names, right? Ether we rename endpoints to something like `startBlock` and `commitBlock` or we rename `BlockExecutionState` to `TransactionExecutionState`? Which one is more logical for you?
@antouhou @jawid-h